### ISSUE TITLE
Revert GitHubWorkflowService target branch to `main`

### ISSUE THREAD

**Issue by howard-e** [w3c/aria-at-app#1457]
#1456 adds support for running macOS 15 with the VoiceOver Bot using a self hosted runner. The workflow runner repository's branch to target this is currently not on `main`. That should be reverted before a new production push.

cc @ChrisC @stalgiag @ccanash

**Comment by howard-e** [comment]
Resolved by #1458
